src/consts.py

```diff
@@ -1,10 +1,40 @@
-import sys
-if sys.platform == 'win32':
+import platform, os
+
+IS_WINDOWS = platform.system().lower() == "windows"
+
+if IS_WINDOWS:
     import winreg
+    REGISTRY_START_PATHS = [winreg.HKEY_CURRENT_USER, winreg.HKEY_LOCAL_MACHINE]
+else:
+    REGISTRY_START_PATHS = []
+
+class GameID:
+    Minecraft = "mc"
+    MinecraftDungeons = "mcd"
+    MinecraftEducationEdition = "mcedu"

```

Comment:
Yeah, that was what I was planning but since the education edition works very differently it would require new logic and I hadn't had time to get to that yet.